src/main/java/uk/gov/hmcts/darts/arm/service/impl/UnstructuredToArmProcessorImpl.java

```diff
@@ -167,12 +171,15 @@ private void generateAndCopyMetadataToArm(ExternalObjectDirectoryEntity armExter
             try {
                 BinaryData metadataFileBinary = fileOperationService.saveFileToBinaryData(archiveRecordFile.getAbsolutePath());
                 armDataManagementApi.saveBlobDataToArm(archiveRecordFileInfo.getArchiveRecordFile().getName(), metadataFileBinary);
-                armExternalObjectDirectory.setStatus(armStatuses.get(ARM_DROP_ZONE));
-                externalObjectDirectoryRepository.saveAndFlush(armExternalObjectDirectory);
-
+                copiedManifestToArm = true;
             } catch (BlobStorageException e) {
-                log.error("Failed to move BLOB metadata for file {} due to {}", archiveRecordFile.getAbsolutePath(), e.getMessage());
-                updateExternalObjectDirectoryStatusToFailed(armExternalObjectDirectory, FAILURE_ARM_MANIFEST_FILE_FAILED);
+                if (e.getStatusCode() == BLOB_ALREADY_EXISTS_STATUS_CODE) {
+                    log.info("Blob already exists {}", e.getMessage());
```

Comment:
Minor thing, but perhaps standardise on either "Blob" or uppercase "BLOB" as elsewhere for consistency in the log messages.